map/framework.cpp

```diff
@@ -359,7 +359,19 @@ Framework::Framework(FrameworkParams const & params)
   : m_startForegroundTime(0.0)
   , m_storage(platform::migrate::NeedMigrate() ? COUNTRIES_OBSOLETE_FILE : COUNTRIES_FILE)
   , m_enabledDiffs(params.m_enableDiffs)
-  , m_bmManager([this]() -> StringsBundle const & { return m_stringsBundle; })
+  , m_bmManager(BookmarkManager::Callbacks([this]() -> StringsBundle const & { return m_stringsBundle; },
```

Comment:
Could you please initialized bookmarks manager after search engine initialization? Otherwise it's unsafe to work with search engine in these lambdas, because it may be not initialized.

Reply:
Перенесла.